pkg/storaged/storage.css

```diff
@@ -678,3 +678,7 @@ td.storage-action {
 .panel-heading {
     position: static;
 }
+
+.storage_alert_action_buttons > button {
```

Comment:
when `StorageButton` is disabled, it is wrapped in `span` element and this rule would not apply:
![afterkk](https://user-images.githubusercontent.com/12330670/74080833-63c24c00-4a48-11ea-9910-69fb26a24f42.png)
Removing `>` fixes this issue and I don't see where it would cause any issue.